packages/block-editor/src/components/list-view/block.js

```diff
@@ -357,6 +357,7 @@ function ListViewBlock( {
 							__experimentalSelectBlock={
 								updateFocusAndSelection
 							}
+							__unstableDisplayLocation="list-view"
```

Comment:
Required in order that `Rename` option is not added within the canvas block options.